src/EditorFeatures/Core/Commands/SortImportsCommandArgs.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.	
+
+using System.Diagnostics.CodeAnalysis;
+using Microsoft.VisualStudio.Text;
+using Microsoft.VisualStudio.Text.Editor;
+
+namespace Microsoft.CodeAnalysis.Editor.Commands
+{
+    /// <summary>	
+    /// Arguments for the Sort Imports command being invoked.	
+    /// </summary>	
+    [ExcludeFromCodeCoverage]
+    internal class SortImportsCommandArgs : CommandArgs
```

Comment:
Why do we have both the old command and new command? We don't need both?

Reply:
(#32623 might point to all this being dead if I'm deleting it there)

Reply:
Removed